pkg/testcase/clusterreset.go

```diff
@@ -0,0 +1,121 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestClusterReset() {
+	killall()
```

Comment:
lets add some logging like 
" oh here we are killing server(S) " idk 

"here we are stopping"


" all good" 

" deleting db "  

etc ...

 ``` go

shared.LogLevel("LEVEL DESIRED ", "your message and args needed ")

```

Reply:
Does this accomplish the logging you are speaking of?

```
•DEBU[2024-04-04T18:57:42Z] Execute: sudo rke2-killall.sh on [redacted]
DEBU[2024-04-04T18:57:50Z] Execute: sudo rke2-killall.sh on [redacted]
DEBU[2024-04-04T18:58:25Z] Execute: sudo systemctl --no-block stop rke2-server on [redacted]
DEBU[2024-04-04T18:58:27Z] StdOut:
DEBU[2024-04-04T18:58:27Z] Execute: sudo systemctl --no-block status rke2-server on [redacted]
DEBU[2024-04-04T18:58:28Z] Execute: sudo rke2 server --cluster-reset on [redacted]
DEBU[2024-04-04T18:59:40Z] Execute: sudo rm -rf /var/lib/rancher/rke2/server/db on [redacted]
DEBU[2024-04-04T18:59:41Z] StdOut:
DEBU[2024-04-04T18:59:41Z] Execute: sudo -i ls -l /var/lib/rancher/rke2/server/db on [redacted]
DEBU[2024-04-04T18:59:42Z] Execute: sudo rm -rf /var/lib/rancher/rke2/server/db on [redacted]
DEBU[2024-04-04T18:59:43Z] StdOut:
DEBU[2024-04-04T18:59:43Z] Execute: sudo -i ls -l /var/lib/rancher/rke2/server/db on [redacted]
DEBU[2024-04-04T18:59:43Z] Execute: sudo systemctl --no-block start rke2-server on [redacted]
DEBU[2024-04-04T18:59:44Z] StdOut:
DEBU[2024-04-04T18:59:44Z] Execute: sudo systemctl --no-block start rke2-server on [redacted]
DEBU[2024-04-04T18:59:45Z] StdOut:
DEBU[2024-04-04T18:59:45Z] Execute: sudo systemctl --no-block start rke2-server on [redacted]
DEBU[2024-04-04T18:59:46Z] StdOut:

PASSED! Test: Verifies Cluster Reset
```

Reply:
sorry it does not, because this is just a log from the function executor (which i dont like it , but its ok ) 

so you as caller, should be logging important steps with useful info as well logging possible errors and warns etc